client/src/components/NavBar.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Button from "@mui/material/Button";
+import { Link } from "react-router-dom";
+
```

Comment:
Theres nothing terribly exciting about this new component, it just uses the MUI setup.